### PR TITLE
fix: missing labels in charts (#257)

### PR DESCRIPTION
**Pull request by halcin**
Update Chart.js version because the version was incompatible with chartjs-plugins-datalabels.

### CODE REVIEW

**Review by fraxken — APPROVED** [review]

File: public/scripts/main.js [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
 // Import Internal Dependencies
-import "../lib/chart.js";
 import { md5 } from "../lib/md5.js";
 
 // Import ESBUILD Assets
 import defaultAvatarURL from "../img/avatar-default.png";
 
 // CONSTANTS
-const kChartOptions = {
+var kChartOptions = {
```
Comment (PierreDemailly) [inline public/scripts/main.js:10]: Why using `var` here ? It seem you can keep `const`, i am missing something?

File: public/scripts/main.js [hunk 2]
```diff
@@ -51,18 +58,27 @@ function* interpolateColors(dataLength, scale, range) {
 function createChart(elementId, type = "bar", payload = {}) {
   const { labels, data, interpolate = d3.interpolateCool } = payload;
   const options = JSON.parse(JSON.stringify(kChartOptions));
+  if (type === "horizontalBar") {
+    type = "bar";
```
Comment (PierreDemailly) [inline public/scripts/main.js:62]: This trigger [no-param-reassign](https://eslint.org/docs/rules/no-param-reassign)

File: public/scripts/main.js [hunk 3]
```diff
@@ -51,18 +58,27 @@ function* interpolateColors(dataLength, scale, range) {
...
+    options.indexAxis = "y";
+  }
   if (type === "pie") {
     options.legend.display = true;
     options.plugins.datalabels.align = "center";
-  }
-  else {
+  } else {
+    options.plugins.legend.display = false;
```
Comment (PierreDemailly) [inline public/scripts/main.js:69]:
```suggestion
  } 
  else {
    options.plugins.legend.display = false;
```

You need to check the linter is OK ;)
There are other small problems, you can run `npx eslint src public` if your IDE is not configured (if you install ESLint extension you should not miss theses errors, i guess you're using VSC)

**Review by PierreDemailly — CHANGES_REQUESTED** [review]

File: public/scripts/main.js (hunk 3 above)
Comment (halcin) [reply public/scripts/main.js]: I I corrected that

**Review by PierreDemailly — APPROVED** [review]

**Pull request merged**